kinesis/kinesis_test.go

```diff
@@ -45,7 +45,7 @@ func newMockOutputPlugin(client *mock_kinesis.MockPutRecordsClient, isAggregate
 		stream:                "stream",
 		client:                client,
 		dataKeys:              "",
-		partitionKey:          "",
+		partitionKey:          "kubernetes->app",
```

Comment:
please add some more test cases:
- partition key with more than one "->"
- "normal" case where the partition key does not container "->"
- error case where the partition key is not found

Reply:
Have already added more test cases for it